runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -90,6 +90,26 @@
     @Getter
     private boolean logHasNoCheckPoint = false;
 
+    private boolean whiteList = false;
+    private List<UUID> streamsToLoad = new ArrayList<>();
+
+    /**
+     * Enable whiteList mode where we only reconstruct
+     * the streams provided through this api.
+     * @param streamsToLoad
+     */
+    public FastObjectLoader addStreamsToLoad(List<String> streamsToLoad) {
```

Comment:
What's the definition of white lists here, only load streams that are in the white list? 
Make sure that they are mutually exclusive (i.e. can't use both lists at the same time).

Reply:
Will add in the comments that they are mutually exclusive. The way to enforce it is by using the whiteList flag.